### PR TITLE
Add some information to the map in aboutus page

### PR DESCRIPTION
**Pull request by jyx11011**
Add name, photo and address to the infoWindow for markers in about us page

### CODE REVIEW

**Review by Jay-kew — CHANGES_REQUESTED** [review]
Hey Yuxin, I think this is good, really gives the map more information! I think you just need to change the indentation to 2 for the `map.css` and it can be merged :)

File: src/main/webapp/js/about-us-map.js [hunk 1]
```diff
@@ -8,20 +8,39 @@ function createDefaultMap(){
...
+}
+
+function infoContent(title, address, url){
+  var infoDiv = document.createElement('div');
+  infoDiv.classList.add('info');
+  var name = document.createElement('h1');
+  name.innerHTML = title;
+  var img = document.createElement('img');
+  img.src = url;
+  var a = document.createElement('p');
```
Comment (calebmah) [inline src/main/webapp/js/about-us-map.js]: nit: could call this var `address` or something more informative

File: src/main/webapp/js/about-us-map.js [hunk 2]
```diff
@@ -8,20 +8,39 @@ function createDefaultMap(){
...
+  a.innerHTML = 'Address: ' + address;
+  infoDiv.appendChild(img);
+  infoDiv.appendChild(name);
+  infoDiv.appendChild(a);
+  return infoDiv;
 }
 
 /* Add landmark to map */
-function createMarker(lat, lng, title, address) {
+function createMarker(lat, lng, title, address, url) {
```
Comment (calebmah) [inline src/main/webapp/js/about-us-map.js]: nit: feel like this should be `image_url` or similar since it needs to be a link to an image

**Review by calebmah — APPROVED** [review]

**Review by calebmah — APPROVED** [review]

**Review by Jay-kew — APPROVED** [review]
I think a few more indentation problems were raised by hound, I'll approve first but just be careful and resolve them before merging! :)

**Pull request merged**